### PR TITLE
migrating moveit_msgs includes

### PR DESCRIPTION
**Pull request by MohmadAyman**
### Description

Please explain the changes you made, including a reference to the related issue if applicable

### Checklist
- [ ] **Required by CI**: Code is auto formatted using [clang-format](http://moveit.ros.org/documentation/contributing/code)
- [ ] Extended the tutorials / documentation, if necessary [reference](http://moveit.ros.org/documentation/contributing/)
- [ ] Include a screenshot if changing a GUI
- [ ] Document API changes relevant to the user in the moveit/MIGRATION.md notes
- [ ] Created tests, which fail without this PR [reference](http://docs.ros.org/kinetic/api/moveit_tutorials/html/doc/tests.html)
- [ ] Decide if this should be cherry-picked to other current ROS branches
- [ ] While waiting for someone to review your request, please consider reviewing [another open pull request](https://github.com/ros-planning/moveit/pulls) to support the maintainers

[//]: # "You can expect a response from a maintainer within 7 days. If you haven't heard anything by then, feel free to ping the thread. Thank you!"

### CODE REVIEW

File: moveit_core/planning_interface/include/moveit/planning_interface/planning_response.h [hunk 1]
```diff
@@ -38,9 +38,9 @@
 #define MOVEIT_PLANNING_INTERFACE_PLANNING_RESPONSE_
 
 #include <moveit/robot_trajectory/robot_trajectory.h>
-#include <moveit_msgs/MoveItErrorCodes.h>
-#include <moveit_msgs/MotionPlanResponse.h>
-#include <moveit_msgs/MotionPlanDetailedResponse.h>
+#include <moveit_msgs/msg/moveIt_error_codes.hpp>
```
Comment (mlautman) [inline moveit_core/planning_interface/include/moveit/planning_interface/planning_response.h]: I believe that this is wrong. With the camelcase translation it should be:
```suggestion
#include <moveit_msgs/msg/move_it_error_codes.hpp>
```

File: moveit_experimental/kinematics_constraint_aware/include/moveit/kinematics_constraint_aware/kinematics_constraint_aware.h [hunk 2]
```diff
@@ -44,9 +44,9 @@
 // ROS msgs
 #include <moveit_msgs/GetConstraintAwarePositionIK.h>
 #include <geometry_msgs/PoseStamped.h>
-#include <moveit_msgs/MoveItErrorCodes.h>
-#include <moveit_msgs/Constraints.h>
-#include <moveit_msgs/RobotState.h>
+#include <moveit_msgs/msg/moveIt_error_codes.hpp>
```
Comment (mlautman) [inline moveit_experimental/kinematics_constraint_aware/include/moveit/kinematics_constraint_aware/kinematics_constraint_aware.h]: see above

File: moveit_experimental/kinematics_constraint_aware/include/moveit/kinematics_constraint_aware/kinematics_request_response.h [hunk 3]
```diff
@@ -45,7 +45,7 @@
 #include <geometry_msgs/PoseStamped.h>
 
 // MoveIt!
-#include <moveit_msgs/MoveItErrorCodes.h>
+#include <moveit_msgs/msg/moveIt_error_codes.hpp>
```
Comment (mlautman) [inline moveit_experimental/kinematics_constraint_aware/include/moveit/kinematics_constraint_aware/kinematics_request_response.h]: Find-replace these.

File: moveit_core/planning_interface/include/moveit/planning_interface/planning_response.h (hunk 1 above)
Comment (MohmadAyman) [reply moveit_core/planning_interface/include/moveit/planning_interface/planning_response.h]: @mlautman  I was puzzled by this too, so moveIt is not considered a single word?

Comment (mlautman) [reply moveit_core/planning_interface/include/moveit/planning_interface/planning_response.h]: It isn't. The auto message generation assumes that the msg names use camel case and then converts it to underscores to create the message header files. This results in a translation from `MoveItSomeMsg.msg` -> `move_it_some_msg.hpp`

Comment (MohmadAyman) [reply moveit_core/planning_interface/include/moveit/planning_interface/planning_response.h]: ah okay :+1: 
I've fixed those and pushed a new commit.

Comment (MohmadAyman) [reply moveit_core/planning_interface/include/moveit/planning_interface/planning_response.h]: @mlautman

**Pull request merged**